.github/workflows/build.yml

```diff
@@ -7,15 +7,15 @@ jobs:
   build-ubuntu:
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
We can maybe also drop versions that are EOL.

Reply:
Yes you are right, I am adding it